test/integration/sequelize/query.test.js

```diff
@@ -130,16 +130,17 @@ describe(Support.getTestDialectTeaser('Sequelize'), () => {
           benchmark: true,
         });
 
-        await sequelize.query('select 1;');
-        expect(logger.calledOnce).to.be.true;
+        await sequelize.query(`select 1${dialect === 'ibmi' ? ' FROM SYSIBM.SYSDUMMY1' : ''};`);
+        console.log(logger);
+        expect(logger.called).to.be.true;
```

Comment:
```suggestion
        expect(logger.calledOnce).to.be.true;
```
Or did you change this on purpose?